vault-kubernetes-sidecar/assets/deployment-04-payrole.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: payrole
```

Comment:
Is this supposed to be spelled "payroll" or is it a pun on roles?

Reply:
The pun is fun.

Reply:
But it can be removed after tomorrow.